src/main/ml-modules/root/config/searchResultsSortConfig.mjs

```diff
@@ -1,19 +1,55 @@
+import op from '/MarkLogic/optic';

```

Comment:
I extended checkPredicates.js to account for this file and verified its output before and after this edit matched in DEV (i.e., no new predicates introduced, with or without typos).  This is not one in the same as the correct predicate specified for each semantic sort binding but they passed my manual scan.